interfaces/api/v1/workbench/monitor.py

```
@@ -42,6 +43,76 @@ class ForeshadowStatsResponse(BaseModel):
+            ):
+                ch.update_tension_dimensions(dims)
+                chapter_repo.save(ch)
+                changed = True
+                logger.info(
+                    "监控接口懒回填张力 novel=%s ch=%s composite=%.1f",
+                    novel_id,
+                    ch.number,
+                    dims.composite_score,
+                )
+            prev_tension = float(dims.composite_score)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)